tests/codegen/scratch_buffer.cpp

```diff
@@ -0,0 +1,24 @@
+#include "common.h"
+
+namespace bpftrace {
+namespace test {
+namespace codegen {
+
+TEST(codegen, big_tuple_scratch_buf)
+{
+  test("kprobe:f { (1, \"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx\") }",
```

Comment:
Maybe these tests should explicitly set the config variable so if we change the default they don't change. And maybe have two codegens (under and over threshold) for each fmt and tuples?

Reply:
+1, testing both paths is good idea

Reply:
Done in latest!